src/main/flight/position.c

```diff
@@ -44,33 +45,27 @@
 #include "sensors/sensors.h"
 #include "sensors/barometer.h"
 
+#ifdef USE_RANGEFINDER
+#include "sensors/rangefinder.h"
+#endif
+
 #include "pg/pg.h"
 #include "pg/pg_ids.h"
 
 static float displayAltitudeCm = 0.0f;
 static bool altitudeAvailable = false;
 
-static float zeroedAltitudeCm = 0.0f;
-static float zeroedAltitudeDerivative = 0.0f;
+static float zeroedGpsAltitudeCm = 0.0f;
```

Comment:
Probably needs some define around this variable.

Reply:
I just don't get it, why these errors are not there when I run make test on my machine

Reply:
To catch before pushing usually use 
```
make test
make all_rev
```

Reply:
Thanks